decentraland/unity-explorer
# chore: emotes tech debt

## What does this PR change?

Added updates from https://github.com/decentraland/unity-explorer/issues/1124 ticket

## How to test the changes?

1. Launch the explorer
2. Play happy path, ensure that wearables and emotes work fine

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
